downstream/modules/platform/con-controller-job-scheduling-improvements.adoc

```diff
@@ -0,0 +1,12 @@
+[id="con-controller-job-scheduling-improvements"]
+
+= Job scheduling improvements
+
+When jobs are created using a schedule, a workflow, the UI or the API, they are first created in _Pending_ state. 
+To determine when and where to run this job, a background task called the *Task Manager* collects all pending and running jobs and determines where capacity is available to run the job. 
```

Comment:
I think *Task Manager* should be _Task Manager_

IBM Style: Words and phrases that are defined or explained should use italics.